n_t_with_s/README.md

```diff
@@ -0,0 +1,170 @@
+
+(it may prove useful to simply put this command in a bash script)
+
+and jobs will be automatically farmed out to the cluster
+
+### Final output
+The current final output is a plot comparing stairwayplot and smc++ estimates of $N(t)$, i.e., 
+`homo_sapiens_Gutenkunst/all_estimated_Ne_{selection}.png` , 
+with selection being 'none', 'all' or applied to the 'exons'.
+
+
+## Parameter Description
+
+`seed` : `<class 'int'>` 
+This sets the seed such that any analysis configuration
+and run can be replicated exactly. 
+
+`population_id` : `<class 'int'>`
+This specifies which population will be plotted.
+
+`num_sampled_genomes_per_population` : `<class 'list'>` 
+This is the haploid number
+of genomes to simulate for an analysis run per population.
+
+`replicates` : `<class 'int'>` The number of replicate simulations to run and 
+analyze. 
+
+`num_sampled_genomes_msmc` : `<class 'str'>` 
+This variable allows the user to run msmc independently on multiple 
+sample sizes (to be specified as a comma-separated string of sample sizes).
+For each of these sample sizes,
+`replicates` number of analyses will be run with `msmc`. All of these numbers should
+not exceed `num_sampled_genomes_per_replicate` because the input files are 
+generated by pruning the tree sequences from the original simulations.
+(We might want to do this for the other two programs, as well?)
+
+`num_msmc_iterations` : `<class 'int'>` The number of iterations for each `msmc` 
+run
+
+`output_dir` : `<class 'str'>` This where all the intermediate files and results
+will be stored. This includes all plots produced, as well as all input/output files
+from the simulations and software runs organized into subdirectories by
+replicate seed.
+
+`species` : `<class 'module'>` from `stdpopsim` such as `stdpopsim.homo_sapiens`.
+Chromosomes, demographics, and recombination maps should be derived from this.
+
+`model` : `<class childclass Model>` 
+This is the class that defines the demography associated with a species. All models
+should inherit from `<class Model>`
+
+`genetic_map` : `<class childclass GeneticMap>` This will define the genetic map
+used for simulations.
+
+`chrm_list` : `<class 'str'>` A string of the chromosome names you would like to simulate,
+separated by commas. All chromosomes simulated will be fed
+as a single input into each analysis by the inference programs, for each replicate.
+Set to "all" to simulate all chromosomes for the genome.
+
+`mask_file` : `<class 'str'>` A string indicating the path to a bed formatted file with
+to be masked out of the analysis.
+
+`dfe_id`: `<class 'str'>` A string indicating what Distribution of Fitness Effects to use.
+See http://sesame.uoregon.edu/~adkern/stdpopsim/doc/catalog.html to check all the possible DFEs
+
+`selection`: `<class 'str'>` A string indicating where the DFE should be applied. So far three
+options are allowed: 'exons', where selection is imposed on exons, 'none', and 'all' –– in which 
```

Comment:
so I don't think you need the `selection` parameter and the `annotation_id` in the config.yaml. Why not just allow for `annotation_id` to take the values `all` or `none` as well and get rid of this one?

Reply:
I will take out the annotation_id part but will leave the selection one. It makes it easier to run each pipeline of selection separately.

Reply:
well I think we want the annotation_id though, right? like imagine we want to run this on different annotations

Reply:
I got it!